scripts/install_qt.sh

```diff
@@ -0,0 +1,52 @@
+#!/usr/bin/env bash
+
+set -e
+set -x
+
+apt install --no-install-recommends -y git
+
+git clone --filter=tree:0 https://github.com/mumble-voip/mumble/ repo
+cd repo
+
+if [ "${MUMBLE_VERSION}" = "latest" ]
+then
+	MUMBLE_VERSION=""
+fi
+
+if ! timestamp=$(git show -s --format=%ct ${MUMBLE_VERSION} 2>/dev/null)
+then
+	echo "Non-existent or malformed MUMBLE_VERSION `${MUMBLE_VERSION}` given, cannot continue!" 1>&2
+	cd ..
+	rm -Rf repo
+	apt -y autoremove --purge git
+	exit 1
+else
+	# 1726713847 refers to the timestamp of the commit that switched Mumble to use Qt6
+	if [ ${timestamp} -ge 1726713847 ]
```

Comment:
Using the timestamp won't work. There will be backport commits to the 1.5 branch that are more recent than the Qt 6 introduction but still the 1.5 branch requires Qt5.

Instead, we'll have to decide based on the used Mumble version. Use Qt 6 for "latest" (or unspecified) and for everything 1.6 and above.